### DISCUSSION TITLE
Does SplitChunksPlugin also split nested `node_modules/` dependencies by `defaultVendors` cacheGroup?

### DISCUSSION THREAD

**Discussion by nyngwang** [webpack/webpack#18045] [Q&A]
Say I have a third-party package installed via `npm` and this package has a nested `node_modules/` folder. Would those dependencies in the nested folder be split into new chunks? Thanks!

**Comment by alexander-akait** [comment]
Yes, all nested `node_modules` will be in the `defaultVendors`

**Reply by nyngwang** [reply]
So there will be only one output bundle file for all vendors, right?

**Reply by alexander-akait** [reply]
Yes, because they don't change often

**Reply by nyngwang** [reply]
@alexander-akait I asked this question because when `A->B->C`, which means `A` imports `B` and `B` imports `C` and, say `A` is an npm package and `B`,`C` are the nested packages, I keep thinking that whether these nested packages `B`,`C` will be in their own chunks instead of in the chunk of `A`. So how does webpack decide not to split more chunks from a bigger chunk, i.e. so webpack will not split out `B`,`C` from the chunk of `A`?

**Reply by alexander-akait** [reply]
You can create a small repo and check it out

**Reply by alexander-akait** [reply]
https://github.com/webpack/webpack/blob/main/lib/config/defaults.js#L1416
